dev_guide/templates.adoc

```diff
@@ -51,41 +51,22 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
-== Labels
-link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template. 
+== Creating a Configuration From a Template
 
-There is also the ability to add labels in the template from the command line.
+You can create a configuration from a template with either the Management Console or the CLI, after first uploading a template to your OpenShift instance.
```

Comment:
@adellape I'm not sure I understand the relevance of the first part of your comment here. The section above is the first and general context section about the relevance of templates. I've rewritten this, so it might make more sense now. Thoughts?

Second part: done.